lib/galaxy/model/__init__.py

```diff
@@ -5307,9 +5405,6 @@ class WorkflowInvocationStep(Dictifiable, RepresentById):
         # FAILED='failed',  TODO: implement and expose
     )
 
-    def update(self):
```

Comment:
This breaks this unit test: https://github.com/galaxyproject/galaxy/blob/dev/test/unit/test_galaxy_mapping.py#L550
I assume this is in response to this comment: https://github.com/galaxyproject/galaxy/pull/9722#discussion_r428079456 ? Is the test outdated? Or do we need `update()` after all? @mvdbeek ?

Reply:
Yeah, that unit test looks like it should be updated. Probably just need to drop that line you referenced.

Reply:
It seems to me this could be moved into a separate PR.  Updating the `WorkflowInvocationStep` class (and related tests) is a non-essential tangent here. I fear it may delay things. My vote is to skip it, get the PR in, and then address this. @jmchilton?